src/libraries/System.Numerics.Tensors/src/System.Numerics.Tensors.csproj

```diff
@@ -2,6 +2,7 @@
   
   <PropertyGroup>
     <TargetFrameworks>$(NetCoreAppCurrent);$(NetCoreAppPrevious);$(NetCoreAppMinimum);netstandard2.0;$(NetFrameworkMinimum)</TargetFrameworks>
+    <TargetFrameworks Condition="'$(DotNetBuildFromSource)' != 'true'">$(TargetFrameworks);net7.0;net6.0</TargetFrameworks>
```

Comment:
I just saw this PR. I don't see an underlying issue that is fixed by this change, what was the motivation to add `net7.0` and `net6.0` TFMs to this library?

.NET 9 dotnet/runtime libraries packages support net8.0 (`NetMinimum`) and net9.0 (`NetCurrent`) only.

Looking at the latest package from main, I see that a warning is emitted at consumption time when targeting net6.0 or net7.0: https://dnceng.visualstudio.com/public/_artifacts/feed/dotnet9/NuGet/System.Numerics.Tensors/overview/9.0.0-alpha.1.23519.15

> System.Numerics.Tensors 9.0.0-alpha.1.23519.15 doesn't support $(TargetFramework) and has not been tested with it. Consider upgrading your TargetFramework to net8.0 or later. You may also set &lt;SuppressTfmSupportBuildWarnings&gt;true&lt;/SuppressTfmSupportBuildWarnings&gt; in the project file to ignore this warning and attempt to run in this unsupported configuration at your own risk.